v1/lib/core/nav/SideNav.js

```diff
@@ -133,6 +146,29 @@ class SideNav extends React.Component {
         <script
           dangerouslySetInnerHTML={{
             __html: `
+            var coll = document.getElementsByClassName('collapsible');
```

Comment:
Heyhey, is there any particular reason you're using var in favor of const/let?

Reply:
The code within `<script>` is shipped as-is to browsers, and not all browsers understand `let`. We stick to ES5 in such situations.

Reply:
Thx for the clarification!